src/components/checkbox/checkbox.scss

```diff
@@ -12,6 +12,12 @@ $checkbox-top: 12px !default;
   }
 }
 
+// Needed for check to display correctly in IE11
+table md-checkbox {
```

Comment:
* this is a global change that is worriesome
*  this needs to be namespace to `._md`:
```css
._md table md-checkbox {

}
```